<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminCenter</class>
 <widget class="QMainWindow" name="AdminCenter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>查看商品列表</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string> 请输入商品名称: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_commodity"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_commodity">
             <property name="text">
              <string> 查看</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <column>
           <property name="text">
            <string>商品ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>商品名</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>底价</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>数量</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>发售日期</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>商品状态</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>查看用户列表</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QWidget" name="widget_user" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string> 请输入用户名称: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_user"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>查看</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget_2">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <column>
           <property name="text">
            <string>用户ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>用户名</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>密码</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>余额</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>用户状态</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>查看订单列表</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableWidget" name="tableWidget_3">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <column>
           <property name="text">
            <string>订单ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>商品ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>卖家ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>买家ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>竞拍时间</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>竞拍价</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>订单状态</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>终止拍卖</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_endAuction">
             <property name="text">
              <string> 确认终止拍卖</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>仿宋</family>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="text">
              <string>竞拍结果显示</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget_4">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <column>
           <property name="text">
            <string>订单ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>商品ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>卖家ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>买家ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>商品名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>竞拍时间</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>竞拍价</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>竞拍结果</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
